### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by amsmyth1**
Created Readme

### CODE REVIEW

**Comment by amsmyth1** [comment]
I recommend reviewing the README here: https://github.com/josephbudina/viewing_party/tree/readme

File: README.md [hunk 1]
```diff
@@ -1,25 +1,151 @@
...
+
+## Deployment
+
+Travis CI was utilized for deployment on this project. Please note this project was developed during the licensing issues with `mimemagic` that persisted on the week of March 29, 2021. Adjustments were made, but as the licensing issues develops changes to gems might need to be implements.
+
+To deploy this application please utilize your own hosting service. The original repo is deployed to Heroku-18.
 
-This is the base repo for the [viewing party project](https://backend.turing.io/module3/projects/viewing_party) used for Turing's Backend Module 3.
+## Applicaiton code
+This application code utilizes MVC pricinples along with the Facade Design Pattern.
```
Comment (klatour324) [inline README.md]: Principles is spelled wrong here

**Review by klatour324 — COMMENTED** [review]
Just have one minor spelling, otherwise this looks good especially with the summary!

**Comment by klatour324** [comment]
It worked!! Going to merge before it decides otherwise 😂

**Pull request merged**